src/main/scala/cromwell/engine/backend/sge/SgeBackend.scala

```diff
@@ -42,14 +42,39 @@ class SgeBackend extends Backend with SharedFileSystem with LazyLogging {
         launchQsub(backendCall) match {
           case (qsubRc, _) if qsubRc != 0 => FailedExecution(new Throwable(s"Error: qsub exited with return code: $qsubRc"))
           case (_, None) => FailedExecution(new Throwable(s"Could not parse Job ID from qsub output"))
-          case (_, Some(sgeJobId)) => pollForSgeJobCompletionThenPostProcess(backendCall, sgeJobId)
+          case (_, Some(sgeJobId)) =>
+            val updateDatabaseWithRunningInfo = updateSgeJobTable(backendCall, "Running", None, Option(sgeJobId))
+            val (executionResult, jobRc) = pollForSgeJobCompletionThenPostProcess(backendCall, sgeJobId)
+            updateDatabaseWithRunningInfo onComplete {
+              case _ =>
+                val completionStatus = statusString(executionResult)
+                val updateDatabaseWithCompletionInfo = updateSgeJobTable(backendCall, completionStatus, Option(jobRc), Option(sgeJobId))
+                updateDatabaseWithCompletionInfo onFailure recordDatabaseFailure(backendCall.call, completionStatus, jobRc)
```

Comment:
I believe the conversations tab shows the original context of the code. Over on the files tab, this comment thread has your comment on line 51. Will let @mcovarr comment further on if his issue was addressed.